internal/test_cases/invalid_command_test_case.go

```diff
@@ -55,5 +55,8 @@ func (t *InvalidCommandTestCase) getExpectedOutput() string {
 }
 
 func (t *InvalidCommandTestCase) getFallbackPatterns() []*regexp.Regexp {
-	return []*regexp.Regexp{regexp.MustCompile(`^(bash: )?` + t.Command + `: (command )?not found$`)}
+	return []*regexp.Regexp{
+		regexp.MustCompile(`^(bash: )?` + t.Command + `: (command )?not found$`),
+		regexp.MustCompile(`ash: ` + t.Command + `: not found$`),
```

Comment:
Let's be more strict? 

```suggestion
		regexp.MustCompile(`^ash: ` + t.Command + `: not found$`),
```